apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/CoverageTransformerV2Test.java

```diff
@@ -576,8 +576,18 @@ private static void verifyCodedExtension(String url, String code, String display
     assertTrue(compare.equalsDeep(ex));
   }
 
-  private static void verifyCoverageStatus() {
-    assertEquals("active", coverage.getStatus().toCode());
+  private static void verifyCoverageStatus(String status) {
+    if (status != null && !"".equals(status)) {
```

Comment:
Generally I think this would be written ` if (status != null && status.length > 0)` just to be slightly easier to read. (Apache common utils also has a nice util StringUtils.isNotBlank too for doing this type of null+empty check)